staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -447,6 +447,9 @@ type Devices struct {
 	// +optional
 	// +listType=atomic
 	GPUs []GPU `json:"gpus,omitempty"`
+	// DownwardMetrics creates a virtio serials for exposing the downward metrics to the vmi.
+	// +optional
+	DownwardMetrics *DownwardMetrics `json:"downwardMetrics,omitempty"`
```

Comment:
This should be added to the v1alpha3 API version too. (they are kept in sync)

Reply:
It looks like the v1alpha3 API version is mapped to v1. Therefore, there's not required to do nothing special to have that spec available in both versions. Nevertheless, thanks for the tip, I wasn't aware of this, it will be useful in future implementations on which you need to explicitly define it on multiple API versions :)